src/pybamm/parameters/parameter_values.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, values):
             for citation in self._dict_items["citations"]:
                 pybamm.citations.register(citation)
 
+    @staticmethod
     def _create_from_bpx(bpx, target_soc):
```

Comment:
It looked like this should have been a static method when I was doing the merge and the tests are still passing, @rtimms can you double check this